<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayrollReportPreview</class>
 <widget class="QWidget" name="PayrollReportPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Preview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cmdPrint">
       <property name="toolTip">
        <string>Send to Printer</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="payroll_resources.qrc">
         <normaloff>:/payroll/images/print-32.png</normaloff>:/payroll/images/print-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdPDF">
       <property name="toolTip">
        <string>Generate PDF</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/PDF.png</normaloff>:/images/PDF.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdZoomOut">
       <property name="toolTip">
        <string>Zoom Out</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="payroll_resources.qrc">
         <normaloff>:/payroll/images/Icon_194.ico</normaloff>:/payroll/images/Icon_194.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdZoomIn">
       <property name="toolTip">
        <string>Zoom In</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="payroll_resources.qrc">
         <normaloff>:/payroll/images/Icon_193.ico</normaloff>:/payroll/images/Icon_193.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Page:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinCurrentPage"/>
     </item>
     <item>
      <widget class="QLabel" name="lblOf">
       <property name="text">
        <string>of 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vLay"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="payroll_resources.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
